### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by ABTanjir** [calcinai/xero-php#647]
Is there any proper documentation about hoe to create invoice with each items?
or how to trigger invoice email?
I must say its very poor documented or i must missing something

**Comment by calcinai** [comment]
You're correct, it is very poorly documented.  Currently the extent of it is What's in the readme and wiki.

If you look at the Xero documentation, all of the models map through in the same pattern, so examples should be transferable between models.

As this is a community project, feel free to submit any changes as you work through.  Unfortunately I don't have the capacity to do that at the moment.
